### ISSUE TITLE
Error message always printed despite no error

### ISSUE THREAD

**Issue by ghilainm** [hazendaz/makeself-maven-plugin#2]
Seems like you always display an error message when executing your plugin. You test if makeself exists and if it doesn't you try to set it executable, which of course fails. Directly after you display an error message despite no error occurred. Seems like a bug to me.

https://github.com/hazendaz/makeself-maven-plugin/blob/d2c3bc9511e754c5b0da11b12fb64b3e9c809a11/src/main/java/com/hazendaz/maven/makeself/MakeselfMojo.java#L411

**Comment by hazendaz** [comment]
@ghilainm Thanks for catching that.  It was wrong in both places that was attempted.  I'm pushing a fix shortly.  I've also added further debug logging so we can more accurately see what is going on.

**Comment by hazendaz** [comment]
@ghilainm The error never did actually display.  I think because of how things work, technically there was nothing wrong because I was going to create a file and flagging it to do certain actions which appear to actually take place when needed.  However, logically it looked bad in the wrong spot so I moved it and that is on the head now.  Also, lots of new debugging output so it is clear things are occurring as expected.  I ran enough prior debugging as well to determine that earlier set was actually working even if order seemed rather strange.  Either way, I think things are better now.  Please take a look and let me know what you think.

**Comment by ghilainm** [comment]
@hazendaz No problem, are you going to deliver soon a new version ?

**Comment by hazendaz** [comment]
Yes.  I'm waiting on some feedback from a Mac user but pretty sure i addressed that issue.  I also want to confirm if new makeself was released yet as I added some newer support off it's master.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by hazendaz** [comment]
As this was non issue and arrangement of code is now more clear as well as I'm pushing a new release where maven is going to be aware now on the shell getting created, I'm closing this. If issues are seen as present after release, please open a new ticket.
